perf/security/mtls/templates/service-graph.yaml

```diff
@@ -0,0 +1,246 @@
+apiVersion: v1
+data:
+  service-graph: |
+    services:
```

Comment:
Since we are just measuring mtls overhead, we don’t need service graph at all.
You should model this on the benchmark test which has fewer moving parts.